apps/api/src/app/workflows-v2/usecases/preview/preview.usecase.ts

```diff
@@ -127,7 +139,9 @@ export class PreviewUsecase {
           preview: executeOutput.outputs as any,
           type: stepData.type as unknown as ChannelTypeEnum,
         },
-        previewPayloadExample: mergedVariablesExample,
+        previewPayloadExample: isEnhancedDigestEnabled
+          ? cleanPreviewExamplePayload(previewPayloadExample)
```

Comment:
For the Preview tab in the Dashboard, we generate the payload example without the `eventCount` nor `events.length` fields.